lib/testUtils/safeChdir.mjs

```diff
@@ -1,14 +1,15 @@
-const { mkdir } = require('node:fs/promises');
-const { fileURLToPath } = require('node:url');
-const { beforeEach, afterEach } = require('@jest/globals');
+import { fileURLToPath } from 'node:url';
+import { mkdir } from 'node:fs/promises';
+
+import { afterEach, beforeEach } from '@jest/globals';
```

Comment:
If you want consistency : https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

Reply:
Thanks for the comment. We have used the ESLint built-in rule [`sort-imports`](https://eslint.org/docs/latest/rules/sort-imports).

See also here: https://github.com/stylelint/eslint-config-stylelint/blob/7bacc6092cf5ae9869e7c8021d891bda49c45c6b/index.js#L110

To be honest, I don't want so much consistency for imports. 😃